tests/unit/rubrik_cdm_test.py

```diff
@@ -0,0 +1,11 @@
+import pytest
+
+
+@pytest.mark.unit
+def test_unit_header(rubrik):
```

Comment:
One should not test internal methods. These should be covered by tests of your public interface, only in very rare cases does it make sense to bypass this ”rule”.  By sticking to this practice we can make sure our encapsulation is sound.